ecs-cli/modules/commands/flags/flags.go

```diff
@@ -157,32 +157,31 @@ const (
 	All                   = "all"
 )
 
-// OptionalRegionAndProfileFlags provides these flags:
-// OptionalRegionFlag inline overrides region
-// OptionalClusterConfigFlag specifies the cluster profile to read from config
-// OptionalProfileConfigFlag specifies the credentials profile to read from the config
-// OptionalAWSProfileFlag specifies the AWS Profile to use for credential information
-func OptionalRegionAndProfileFlags() []cli.Flag {
+func OptRegionFlag() []cli.Flag {
 	return []cli.Flag{
 		cli.StringFlag{
 			Name: RegionFlag + ", r",
 			Usage: fmt.Sprintf(
 				"[Optional] Specifies the AWS region to use. Defaults to the region configured using the configure command",
 			),
 		},
-		cli.StringFlag{
-			Name: ClusterConfigFlag,
-			Usage: fmt.Sprintf(
-				"[Optional] Specifies the name of the ECS cluster configuration to use. Defaults to the default cluster configuration.",
-			),
-		},
+	}
+}
+
+func OptECSProfileFlag() []cli.Flag {
```

Comment:
Since the other functions still start with optional, how about all of these start with optional as well, just for consistency (or have all of them start with "Opt"